.github/workflows/test_workflows.yml

```diff
@@ -19,32 +19,10 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
 
-  build_image_amd64:
-    name: build_image_amd64
-    needs: checks
+  e2e:
+    name: e2e
```

Comment:
e2e covers image build verification, so now we only need two checks:
1. lint and unit test
2. E2E (bvt) test